packages/flutter/test/material/text_selection_theme_test.dart

```diff
@@ -69,52 +71,12 @@ void main() {
     await tester.pumpAndSettle();
     final EditableTextState editableTextState = tester.firstState(find.byType(EditableText));
     final RenderEditable renderEditable = editableTextState.renderEditable;
-    expect(renderEditable.cursorColor, const Color(0x004285f4));
-    expect(renderEditable.selectionColor, const Color(0xFF90CAF9));
-
-    // Test the selection handle color.
-    await tester.pumpWidget(
-      MaterialApp(
-        home: Material(
-          child: Builder(
-            builder: (BuildContext context) {
-              return materialTextSelectionControls.buildHandle(
-                  context, TextSelectionHandleType.left, 10.0
-              );
-            },
-          ),
-        ),
-      ),
-    );
-    await tester.pumpAndSettle();
-    final RenderBox handle = tester.firstRenderObject<RenderBox>(find.byType(CustomPaint));
-    expect(handle, paints..path(color: Colors.blue[300]));
-
-  });
-
-  testWidgets('Empty textSelectionTheme with useTextSelectionTheme set will use new defaults', (WidgetTester tester) async {
-    final ThemeData theme = ThemeData.fallback().copyWith(useTextSelectionTheme: true);
-    final Color primaryColor = Color(theme.colorScheme.primary.value);
-
-    // Test TextField's cursor & selection color.
-    await tester.pumpWidget(
-      MaterialApp(
-        theme: theme,
-        home: const Material(
-          child: TextField(),
-        ),
-      ),
-    );
-    await tester.pumpAndSettle();
-    final EditableTextState editableTextState = tester.firstState(find.byType(EditableText));
-    final RenderEditable renderEditable = editableTextState.renderEditable;
     expect(renderEditable.cursorColor, primaryColor.withAlpha(0));
-    expect(Color(renderEditable.selectionColor.value), primaryColor.withOpacity(0.12));
+    expect(Color(renderEditable.selectionColor.value), primaryColor.withOpacity(0.40));
```

Comment:
Nit: Is there any way this could be a constant?  Just thinking that it's not intuitive where this value comes from by looking at the test (I see it hard coded in material/text_field.dart).

Don't worry about this if it's not a quick and obvious improvement, I know it's not really this PR's responsibility!